packages/editor/src/components/collapsible-block-toolbar/index.js

```diff
@@ -24,24 +24,18 @@ import { unlock } from '../../lock-unlock';
 
 const { useHasBlockToolbar } = unlock( blockEditorPrivateApis );
 
-export default function CollapsibleBlockToolbar( { isCollapsed, onToggle } ) {
+export default function CollapsibleBlockToolbar() {
 	const { blockSelectionStart } = useSelect( ( select ) => {
 		return {
 			blockSelectionStart:
 				select( blockEditorStore ).getBlockSelectionStart(),
 		};
 	}, [] );
 	const hasBlockToolbar = useHasBlockToolbar();
+	const [ isCollapsed, setIsCollapsed ] = useState( false );
 
 	const hasBlockSelection = !! blockSelectionStart;
 
-	useEffect( () => {
-		// If we have a new block selection, show the block tools
-		if ( blockSelectionStart ) {
-			onToggle( false );
-		}
-	}, [ blockSelectionStart, onToggle ] );
```

Comment:
Oh, I think we'll want to keep this. It should uncollapse whenever you change selection I guess.